### PR TITLE
Adds aldryn-events

### PR DESCRIPTION
**Pull request by FinalAngel**
- aldryn-events configs
- improve config files

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -66,22 +65,22 @@ gulp.task('tests:webdriver', webdriverUpdate);
 gulp.task('tests:integration', ['tests:webdriver'], function () {
     return gulp.src([PROJECT_PATH.tests + '/integration/*.js'])
         .pipe(protractor({
-            configFile: PROJECT_PATH.tests + '/protractor.conf.js',
-            args: []
+            'configFile': PROJECT_PATH.tests + '/protractor.conf.js',
```
Comment (vxsx) [inline gulpfile.js]: i would rather get rid of them

Comment (FinalAngel) [inline gulpfile.js]: you mean `configFile` instead of `'configFile'`

Comment (vxsx) [inline gulpfile.js]: yes exactly, thank you ^^

**Comment by FinalAngel** [comment]
Still waiting for aldryn-events release to unpin `-e git://github.com/aldryn/aldryn-events.git#egg=aldryn-events`

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**